list.rb

```diff
@@ -0,0 +1,25 @@
+require './enumerable'
```

Comment:
- [ ] I would recommend using ` require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."